app/templates/serviceLearning/slcNewProposal.html

```diff
@@ -33,6 +33,7 @@
     </div>
     <div class="col" align="right">
       <button class="btn btn-primary" type="button" id="nextButton">Next</button>
+      <button class="btn btn-success" type="button" id="approveButton">Approve</button>

```

Comment:
Button name should indicate that we are also saving the form. Maybe "Save and Approve"